tests/test_utils.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2015 Confluent Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import socket
+
+
+def find_available_port(min_port=8000, max_port=9000):
```

Comment:
This actually isn't a reliable way of getting a port -- once you close a socket you cannot be sure of its availability. This was the source of a bunch of issues in the Kafka unit tests.

It may not matter much for the couple of tests we have here, just beware that this can be a problem and can lead to confusing test failures.